scraper/gosnmp.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018 The Prometheus Authors
```

Comment:
Nit, new files should have the current year.

```suggestion
// Copyright 2024 The Prometheus Authors
```